examples/csharp/documentation/ProviderClassesStack.cs

```diff
@@ -36,10 +36,24 @@ public ProviderClassesStack(Construct scope, string name) : base(scope, name)
                 InstanceType = "t2.micro"
             });
 
+            TerraformVariable dnsimpleToken = new TerraformVariable(this, "dnsimpleToken", new TerraformVariableConfig
+            {
+                Type = "string",
+                Description = "dnsimple token",
+                Sensitive = true
+            });
+
+            TerraformVariable dnsimpleAccount = new TerraformVariable(this, "dnsimpleAccount", new TerraformVariableConfig
+            {
+                Type = "string",
+                Description = "dnsimple account",
+                Sensitive = true
+            });
+
             new DnsimpleProvider(this, "dnsimple", new DnsimpleProviderConfig
             {
-                Token = Token.AsString(Environment.GetEnvironmentVariable("DNSIMPLE_TOKEN")),
-                Account = Token.AsString(Environment.GetEnvironmentVariable("DNSIMPLE_ACCOUNT"))
+                Token = Token.AsString(dnsimpleToken.StringValue),
+                Account = Token.AsString(dnsimpleAccount.StringValue)
```

Comment:
I think the `Token.AsString()` bit can be removed, as `.StringValue` is doing that internally already :)

Reply:
👍